### PR TITLE
support loading properties file in unnamed resources module.

### PR DESCRIPTION
**Pull request by arjenzhou**
#1826

### CODE REVIEW

**Comment by arjenzhou** [comment]
ping @brettwooldridge

File: src/main/java/com/zaxxer/hikari/HikariConfig.java [hunk 1]
```diff
@@ -18,17 +18,14 @@
...
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.naming.InitialContext;
 import javax.naming.NamingException;
 import javax.sql.DataSource;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
+import java.io.*;
```
Comment (sivaindev) [inline src/main/java/com/zaxxer/hikari/HikariConfig.java:28]: Consider replacing the wildcard import `import java.io.*;` with explicit imports for only the classes used in this file. This improves readability, makes dependencies clearer during code review, and avoids potential future naming collisions.

**Comment by gdspw** [comment]
谢谢你的来信，目前有点忙，稍后会尽快回复您！

**Comment by sivaindev** [comment]
> #1826

https://github.com/brettwooldridge/HikariCP/issues/1826#issuecomment-4361007359

**Pull request merged**